internal/provider/auth_token_file_test.go

```diff
@@ -147,38 +144,6 @@ func TestAuthLoginTokenFile_Login(t *testing.T) {
 			},
 			wantErr: false,
 		},
-		{
```

Comment:
The Auth Token File directly copies a token from file and sets it on a cloned client. It does not use the common `AuthLoginCommon` method which performs the token validation. This test is unrelated for the Auth Token File Login flow, hence removed